src/models/focus.js

```diff
@@ -99,9 +100,19 @@ nv.models.focus = function(content) {
             brush
                 .x(x)
                 .on('brush', function() {
-                    onBrush();
+                    onBrush(syncBrushing);
                 });
 
+            if (!syncBrushing) {
+                brush.on('brushend', function () {
+                    var extent = brush.empty() ? x.domain() : brush.extent();
+                    if (Math.abs(extent[0] - extent[1]) <= 1) {
+                        return;
+                    }
+                    dispatch.onBrush(extent);
```

Comment:
No, onBrush(extent) will pass the 'extent' to LineChart and StackedAreaChart to determine if a value in the dataSet will be filtre or not. Please check the onBrush() function defined in LineChart and StackedAreaChart.